src/main/scala/FileSystem.scala

```diff
@@ -0,0 +1,34 @@
+
+
+class FileSystem {
+  def commandInterpretor() : Unit = {
+    print( "Your cmd (press \"q\" to quit, \"h\" for help): " )
+    val inputKeyboard = getKeyboardInput
+
+    val cmdMatcher = inputKeyboard match{
+      case "q" => "The key q is pressed, the app will close."
+      case "h" =>  help(); commandInterpretor()
+      case "curdir" => println( getCurrentPath ); commandInterpretor()
+      case _   => println("The command is not recognized. Type \"h\" to list the commands and how to use them or \"q\" to quit."); commandInterpretor()
+    }
+
+    println( cmdMatcher )
+  }
+
+  def help() : Unit = {
+    println(
+      "Run the following commands (without quotes):\n" +
+        "\"curdir\" = will show the current directory"
```

Comment:
Same here. You can use triple quotes for better and clearer formatting
```suggestion
      """Run the following commands (without quotes):
        |"curdir" = will show the current directory
        |""".stripMargin
```

Reply:
> Nice start! I like that you created an interpretor for your commands, which is more functional-like. The only remark i would have is that the idea of the exercise is to model the filesystem and it's commands, so it all should be a simulation which doesn't use the SO filesystem. Using `java.io.File` is more of a cheat. I recommend creating models for Files/Directories and also for the commands you used.

Thank you, I will re-think and re-format to be in Simulation-like and not real file-handling.